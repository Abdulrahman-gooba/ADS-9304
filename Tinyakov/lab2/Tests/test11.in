(cos(*(45)(x)))x
