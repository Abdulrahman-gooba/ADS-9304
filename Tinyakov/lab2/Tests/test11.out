-sin((45)*(x))*(((0)*(x)+(45)*(1)))
